.kitchen.yml

```diff
@@ -1,12 +1,13 @@
 ---
 driver:
-  name: vagrant 
+  name: vagrant
 
 provisioner:
-  name: chef_solo
+  name: chef_zero
```

Comment:
Can you revert this one change? There's no reason to switch provisioner, and I prefer testing against solo, since it'll find issues that arise in that environment that don't arise in a `chef-zero/chef-client` setup.